allure-junit5/src/test/java/io/qameta/allure/examples/junit5/AllureQuickStartTest.java

```diff
@@ -7,7 +7,7 @@
 
 import static io.qameta.allure.Allure.step;
 
-public class AllureSimpleTest {
+public class AllureQuickStartTest {
 
```

Comment:
The main question here is do we list here all usuful for an average user and removed not useful for an average user? 

The quick start should probably be the place for all "snippets" most often used by a regular user, also counting some common best practices... 

Thinking this way, I would add here a few tests... Mark/label/tag one as belonging to smoke test suite. Another would make failing and mark corrspondingly, adding links to some bugs... 

Maybe we can consider something else...